cli/azd/cmd/testdata/TestUsage-azd-config-get.snap

```diff
@@ -1,12 +1,18 @@
-Usage:
-  azd config get <path> [flags]
 
-Flags:
-  -h, --help            Gets help for get.
-  -o, --output string   The output format (the supported formats are json). (default "json")
+Gets a configuration
+
+Usage
+  {{if .Runnable}}{{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}{{.CommandPath}} [command]{{end}}
```

Comment:
I would maybe expect this to be the actual substituted values instead of template parameters. Do we need to update the test to produce this?

Reply:
The strategy is actually to set the cobra's `help template`.  We are testing that the template is as we expect.

Reply:
I have updated the tests to resolve the template. This will help to test both, the template and that the template is valid.